source/components/Input.js

```diff
@@ -1,39 +1,48 @@
+// @flow
 import React, { Component } from 'react';
-import { bool, func, object, number, string, shape } from 'prop-types';
-import { withTheme } from '../themes/withTheme';
-
-// external libraries
+// $FlowFixMe
+import type { ComponentType, Element, SyntheticInputEvent } from 'react';
```

Comment:
Would be great if you could a bit of context why `$FlowFixMe` was necessary here!

Reply:
Same as below, just copying the comment incase you read this one first: These types are the recommended way to type DOM events with React and were released v16.3-ish, but Flow doesn't recognize them as part of the React module yet.